frontend/src/scenes/data-management/event-properties/eventPropertyDefinitionsTableLogic.ts

```diff
@@ -143,6 +144,29 @@ export const eventPropertyDefinitionsTableLogic = kea<
                 )
             )
         },
+        loadEventPropertyDefinitionsSuccess: () => {
+            if (cache.propertiesStartTime !== undefined) {
+                eventUsageLogic
+                    .findMounted()
+                    ?.actions.reportDataManagementEventPropertyDefinitionsPageViewed(
+                        performance.now() - cache.propertiesStartTime,
+                        values.eventPropertyDefinitions.results.length
+                    )
+                cache.propertiesStartTime = undefined
+            }
+        },
+        loadEventPropertyDefinitionsFailure: ({ error }) => {
+            if (cache.propertiesStartTime !== undefined) {
+                eventUsageLogic
+                    .findMounted()
+                    ?.actions.reportDataManagementEventPropertyDefinitionsPageViewed(
+                        performance.now() - cache.propertiesStartTime,
+                        -1,
+                        error ?? 'There was an unknown error fetching property definitions.'
```

Comment:
Same thought on `-1` being odd here. And breaking this event into 2: `...loaded` and `...success`